x-pack/plugins/security/public/authentication/login/_login_page.scss

```diff
@@ -27,3 +27,7 @@
     max-width: 700px;
   }
 }
+
+.loginWelcome__selectorButton {
+  margin-bottom: $euiSize;
```

Comment:
FYI, instead of using `margin`, you can use [`EuiSpacer`](https://elastic.github.io/eui/#/layout/spacer)

Reply:
Thanks, I was torn on this and decided to go with CSS solution. But I don't have strong opinion here. Do we recommend using `EuiSpacer` in cases like this?

Reply:
Yes you can find a link to the docs in the comment above. Thanks!

Reply:
Hmm, actually I use this margin for the elements that are rendered in the loop, but it seems we don't recommend using spacer in such case:
> You should not use it in loops of repeatable components. In those situations it is almost always more preferable to define the spacing on the component itself.

I'll see if I can turn the element I render into component.

Reply:
Yeah sorry that is an old warning. I actually have a branch going that will remove that message from our docs. It **is** ok to use EuiSpacer in repeatable loops.

Reply:
Got it, replaced with `EuiSpacer`, thanks!